### PR TITLE
update bukkit dependency to 1.13

### PR DESCRIPTION
**Pull request by RoboMWM**
Unsure for some of the material conversions for fish.

Also there may be some caveats to using IDs now as I experienced with the ExtraHardMode plugin (though the use of it is very light here, just a couple times in a single class).

### CODE REVIEW

**Comment by RoinujNosde** [comment]
> Unsure for some of the material conversions for fish.

Why?

**Comment by RoboMWM** [comment]
The 1.13 update involved a major change as to how fish are classified, so I'm not sure if I got all of the "fish" types or not, and/or if they were bundled into subtypes of a material.

**Comment by RoinujNosde** [comment]
You left these 3 out:
COD, COOKED_COD and TROPICAL_FISH

**Comment by RoboMWM** [comment]
Yea, and I'm not sure what their point/saturation levels are. I guess I could try making guesses from the MC wiki.

**Comment by RoboMWM** [comment]
Thanks. I'm not sure what their point/saturation levels are though. I guess I could try making guesses from the MC wiki.

File: src/main/resources/plugin.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ name: SimpleClans
 main: net.sacredlabyrinth.phaed.simpleclans.SimpleClans
 version: ${project.version}
 softdepend: [VanishNoPacket, Permissions, PermissionsBukkit, Vault, PermissionsEx, mChat, mChatSuite]
+api-version: '1.14'
```
[the remark this answers is not in the document]
Comment (RoboMWM) [reply src/main/resources/plugin.yml]: Yes, as you did not specify 1.13 or 1.14, I defaulted to latest (1.14).

I can set dependency for 1.13 instead if you want.

[the remark this answers is not in the document]
Comment (RoinujNosde) [reply src/main/resources/plugin.yml]: Change it to 1.13, please.

[the remark this answers is not in the document]
Comment (RoinujNosde) [reply src/main/resources/plugin.yml]: I'm gonna test it then, but from what I've researched I think it prevents the plugin from loading.

**Comment by FrozenBeard** [comment]
Hopefully this wont break 1.12.2 support :)

**Comment by RoboMWM** [comment]
This would, as it's using the new material enums for fish.

**Comment by FrozenBeard** [comment]
What do fish have to do with SimpleClans? Does not seem worth breaking the plugin for fish?

**Comment by RoboMWM** [comment]
Well it's not just fish, older materials have been updated to the renamed 1.13 types. As for fish, Simpleclans has the vitals command which includes food count and other stats.

**Comment by RoboMWM** [comment]
Either way I PR'd because the maintainer was interested in an update. This also would allow for PRs that utilize new APIs.

**Comment by RoboMWM** [comment]
That would make for some ugly code. IMO plugins should be updated with maintained versions of MC, which I believe this plugin has done in the past. If 1.12.2 development must be maintained by this plugin, then it could be done in a separate branch. 

But afaik there aren't any major bugs in SimpleClans for 1.12.2, so just having an older version available for an older version of the server makes sense to me.

File: src/main/resources/plugin.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (RoinujNosde) [reply src/main/resources/plugin.yml]: Captura de tela de 2019-09-01 08-14-53

**Comment by RoinujNosde** [comment]
I'm not sure why this wasn't closed automatically :confused:

**Comment by RoboMWM** [comment]
How did you pull it?

**Comment by RoboMWM** [comment]
There's no authorship in the commit, so you must've somehow pulled my changes on your local machine, but not the commit - or you somehow squashed the commits and list the owner.

Github can auto-squash and merge from the web interface here, no need to do all that manually.

**Comment by RoinujNosde** [comment]
I used git pull on your branch.
Then used git merge --squash.

> Github can auto-squash and merge from the web interface here, no need to do all that manually.
I wanted to test a few things first.

**Comment by RoboMWM** [comment]
Ya, you destroyed the authorship doing that. Many ways to remedy that, including --amend.

If you want to make changes, you can make changes to my branch - Github by default allows the maintainers to modify the origin branch of a pull request.

**Comment by RoboMWM** [comment]
Here's an example where I was resolving a merge conflict: https://github.com/TechFortress/GriefPrevention/pull/294#commits-pushed-03f2ff5

**Comment by RoinujNosde** [comment]
I'm new to this git universe.
So I use --amend, and then git merge without --squash?

**Comment by RoboMWM** [comment]
`--amend` is to edit the last commit (i.e. the HEAD) - so you can change the author. e.g. `git commit --amend --author="Author Name <email@address.com>"` to modify author of last commit. You'll need to force push with `git push -f` after doing that since you've already pushed the commit before.

I did a quick search - doesn't seem to be any way to retain author when performing a squash. I probably did an amend like the above.

Alternatively you can just checkout my branch and make the edits there (and push them), and then you can come back to GitHub and pull the changes so it's recognized here, that way you won't have to worry too much about amending and rebasing.

**Comment by RoinujNosde** [comment]
Done.
Thanks for the help!

**Pull request closed without merging**